src/app/home.component.html

```diff
@@ -16,6 +6,12 @@
   showTableOfContents="true"
   [sidebarRoutes]="sidebarRoutes">
 
+  <stache-internal>
```

Comment:
This I think needs to be removed?  I think we removed the internal stuff from Stache into the restricted view library